content/blog/2022/headed-playwright-in-docker.md

```diff
@@ -189,11 +206,29 @@ services:
 
 Remember: `environment` needs `DISPLAY=:0` in Windows and
 `DISPLAY=host.docker.internal:0` in macOS. After editing and saving the file,
-run `docker-compose` to open the browser (or run any Playwright command) along
-with the Playwright Inspector:
+open the browser along with the Playwright Inspector with this command:
 
 ```bash
-docker-compose run web npx -y playwright open google.com
+docker compose run --rm web npx -y playwright open google.com
+```
+
+## Debugging an Existing Test Suite
```

Comment:
Same question here -- since these are the two main additions, do we want to use `{% callout 'note' %}` instead of just updating the text?

cc @mirisuzanne @SondraE I'm not sure what the trade-offs are for that decision.

Reply:
This makes me wonder, do we have a way to track created and updated dates separately in the meta data?

Reply:
I don't think we do currently -- Sondra asked about that [here](https://github.com/oddbird/oddleventy/pull/563#pullrequestreview-2020719915)

Reply:
I've been putting updates in a note. I think it's good to have dates associated with changes.

Reply:
@mirisuzanne @jerivas @jgerigmeyer I saw that Mia did some of each. I'm not quite sure how she decided when to callout and when not. 
I would say that until we have the new metadata that says "last update April 24, 2024" I would lean heavily toward using the callout note at the top of the article with the date.

Reply:
I made some 'editorial' changes outside the update, where I just felt like a small adjustment to the language would be more accurate - and then used callout boxes where I wanted to clarify some more substantial change.

Reply:
Added a note in 7327848f871ccc9ffa1e5be256f2c978bba8446a